src/allmydata/storage_client.py

```diff
@@ -80,11 +80,23 @@ def __init__(self, permute_peers, preferred_peers=(), tub_options={}, tub_handle
         # own Reconnector, and will give us a RemoteReference when we ask
         # them for it.
         self.servers = {}
-        self.static_servers = []
+        self._static_server_ids = set() # ignore announcements for these
         self.introducer_client = None
         self._threshold_listeners = [] # tuples of (threshold, Deferred)
         self._connected_high_water_mark = 0
 
+    def set_static_servers(self, servers):
+        for (server_id, server) in servers.items():
+            self._static_server_ids.add(server_id)
+            handlers = self._tub_handlers.copy()
+            handlers.update(server.get("connections", {}))
```

Comment:
yeah, I wanted it to match the tahoe.cfg section which controls the defaults. So tahoe.cfg has:

```
[connections]
 tcp = tor
```

and `servers.yaml` has:

```
storage:
  v0-c2ng2pbrmxmlwpijn3mr72ckk5fmzk6uxf6nhowyosaubrt6y5mq:
    ann:
      nickname: my-server-1
      anonymous-storage-FURL: pb://u33m4y7klhz3bypswqkozwetvabelhxt@tcp:10.1.2.3:51298/eiu2i7p6d6mm4ihmss7ieou5hac3wn6b
      permutation-seed-base32: w2hqnbaa25yw4qgcvghl5psa3srpfgw3
    connections:
      # this overrides the tcp=tor from tahoe.cfg, for just this server
      tcp: tcp
```